guidelines/program-guidelines.md

```diff
@@ -0,0 +1,81 @@
+# Program Guidelines
+
+## Contributors
+
+- Ashley
+- Jorge
+- Ricky
+- Manish
+
+## Have a clear audience goal
+
+Before you review talks, have a clear idea of what your target audience is. This is not just “users of the technology the conference is about”, this is also includes subgroups by field or level of expertise. Beginner talks will bore advanced users, and advanced talks will not be useful for beginners. When you have multiple rooms, having multiple targets is more feasible; however this can be tricky to schedule, so be careful.
```

Comment:
"and advanced talks will not be useful for beginners"
I think it might be ok to soften this one. There are advanced talks that can incite interest in beginners.